### PR TITLE
Implement AOS-62 WriteMultipart returns Part

### PR DESCRIPTION
**Pull request by JinnyYi**
ref: https://github.com/aos-dev/go-storage/issues/571.

### CODE REVIEW

File: multiparter.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ func TestMultiparter(t *testing.T, store types.Storager) {
 			size := rand.Int63n(4 * 1024 * 1024) // Max file size is 4MB
 			r := io.LimitReader(randbytes.NewRand(), size)
 
-			n, err := m.WriteMultipart(o, r, size, 0)
+			n, _, err := m.WriteMultipart(o, r, size, 0)
```
Comment (Xuanwo) [inline multiparter.go]: We need to test returning part is not nil.

Comment (JinnyYi) [reply multiparter.go]: Do we need to check part.Size?

Comment (Xuanwo) [reply multiparter.go]: We need an extra proposal to define this behavior, let's just skip it.

**Review by Xuanwo — APPROVED** [review]
